litmus-portal/frontend/src/views/WorkflowDetails/WorkflowStatus/index.tsx

```diff
@@ -28,16 +29,14 @@ const WorkflowStatus: React.FC<WorkflowStatusProps> = ({ phase }) => {
     <div className={classes.status}>
       <span className={classes.icon}>
         <img
-          className={
-            phase.toLowerCase() === 'running' ? classes.runningSmallIcon : ''
-          }
+          className={phase === RUNNING ? classes.runningSmallIcon : ''}
           src={`/icons/${phase.toLowerCase()}.svg`}
           alt="status"
         />
       </span>
       <Typography>
         <span className={getStatus(phase)}>
-          <strong>{phase}</strong>
+          <strong>{phase === SUCCEEDED ? 'Completed' : phase}</strong>
```

Comment:
use translation and can you keep the fontWeight in the style class which already is applied to the `span` ?It will help in reducing the code, removing unnecessary jsx.